src/test/scala/org/dianahep/sparkroot/testfirst.scala

```diff
@@ -0,0 +1,781 @@
+package org.dianahep.sparkroot
```

Comment:
Right now we do not anymore do anything with org.dianahep.sparkroot
Use __org.dianahep.sparkroot.experimental__ instead